nixos/modules/services/networking/pixiecore.nix

```diff
@@ -0,0 +1,102 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.pixiecore;
+
+in
+
+{
+
+  options = {
+
+    services.pixiecore = {
+
+      enable = mkEnableOption "Pixiecore";
+
+      debug = mkOption {
+        type = types.bool;
+        default = false;
+        description = "Log more things that aren't directly related to booting a recognized client";
+      };
+
+      logTimestamps = mkOption {
+        type = types.bool;
+        default = true;
+        description = "Add a timestamp to each log line";
+      };
+
+      dhcpNoBind = mkOption {
+        type = types.bool;
+        default = false;
+        description = "Handle DHCP traffic without binding to the DHCP server port";
+      };
+
+      bootMsg = mkOption {
+        type = types.nullOr types.string;
+        default = "";
```

Comment:
Maybe default should be `null`.

Reply:
Looks like this option is never actually used, so I'd vote for just deleting the `mkOption` block.

Reply:
Good catch.